baseplate/lib/experiments/__init__.py

```diff
@@ -15,6 +15,7 @@
 from baseplate.lib.events import EventLogger
 from baseplate.lib.experiments.providers import parse_experiment
 from baseplate.lib.experiments.providers.base import Experiment
+from baseplate.lib.file_watcher import DEFAULT_FILEWATCHER_BACKOFF
```

Comment:
Importing this default all over and setting it as the default seems weird to me, I think a cleaner way to do it would be to always default to `None` and in the actual file_watcher code have something like:

```python
backoff = backoff or 0.01
```

Reply:
Done.